pkg/services/authz/zanzana/zanzana.go

```diff
@@ -8,34 +8,30 @@ import (
 )
 
 const (
-	TypeUser string = "user"
-	TypeTeam string = "team"
+	TypeUser      string = "user"
+	TypeTeam      string = "team"
+	TypeFolder    string = "folder"
+	TypeDashboard string = "dashboard"
 )
 
 const (
 	RelationTeamMember string = "member"
 	RelationTeamAdmin  string = "admin"
+	RelationParent     string = "parent"
 )
 
-func NewObject(typ, id string) string {
-	return fmt.Sprintf("%s:%s", typ, id)
+func NewObject(typ, id, relation string) string {
```

Comment:
We should add a comment describing what relation does here and what an empty relation means